manifests/lia.pp

```diff
@@ -18,10 +18,10 @@
 
   # Setting environment variables is not supported by package
   # but this is needed for setting the LIA password
-  exec {"yum install -y 'EMC-ScaleIO-lia${real_version}'":
+  exec {"/usr/bin/yum install -y 'EMC-ScaleIO-lia${real_version}'":
```

Comment:
you should set this path globally https://www.puppetcookbook.com/posts/set-global-exec-path.html